test/integration_tests/RBF/test_RBF_00004.cpp

```diff
@@ -0,0 +1,141 @@
+/*---------------------------------------------------------------------------*\
+ *
+ *  bitpit
+ *
+ *  Copyright (C) 2015-2021 OPTIMAD engineering Srl
+ *
+ *  -------------------------------------------------------------------------
+ *  License
+ *  This file is part of bitpit.
+ *
+ *  bitpit is free software: you can redistribute it and/or modify it
+ *  under the terms of the GNU Lesser General Public License v3 (LGPL)
+ *  as published by the Free Software Foundation.
+ *
+ *  bitpit is distributed in the hope that it will be useful, but WITHOUT
+ *  ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ *  FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public
+ *  License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with bitpit. If not, see <http://www.gnu.org/licenses/>.
+ *
+\*---------------------------------------------------------------------------*/
+
+#include <iostream>
+#include <vector>
+#include <array>
+#include <algorithm>
+#if BITPIT_ENABLE_MPI==1
+#include <mpi.h>
+#endif
+
+#include "bitpit_RBF.hpp"
+
+using namespace bitpit;
+
+//Description. Directly Interpolate a scalar (z-displacement) field
+
+/*!
+* Subtest 001
+*
+* Testing field interpolation.
+*/
+int subtest_001()
+{
+    // Variables
+    std::vector< std::array<double,3> >         controlNodes;
+    std::vector< std::array<double,3> >         points;
+	std::vector< double >         				zDispl ;
+	
+    // fill control points
+    int nCP = 4;
+    controlNodes.resize(nCP);
+    controlNodes[0] = std::array<double, 3>({0., 0., 0.});
+    controlNodes[1] = std::array<double, 3>({0., 2., 0.});
+    controlNodes[2] = std::array<double, 3>({2., 2., 0.});
+    controlNodes[3] = std::array<double, 3>({2., 0., 0.});
+
+    // fill displacements of control points
+	zDispl.resize(nCP);
+    zDispl[0] = 1.;
+    zDispl[1] = 1.;
+    zDispl[2] = 2.;
+    zDispl[3] = 1.;
+
+    
+     //RBF used to directly interpolate zDispl field
+    bitpit::RBF   paraMorph;
+	RBFBasisFunction funct = RBFBasisFunction::WENDLANDC2;
```

Comment:
@edoardolombardi just a question: I am not sure, so I ask here, isn't the usage of the polynomials more reasonable only when you make use of thin splines (see http://www.scholarpedia.org/article/Radial_basis_function#Further_example_with_linear_additional_terms in the section of polynomias)? If you agree, would you add a thin spline to bitpit, of the form:
f(r)= r ^2 * log(r)? 
If you decide to do it within this PR, maybe line 69 can be substituted?

Reply:
Added in https://github.com/optimad/bitpit/pull/434/commits/b5c33c96b33e7eeb4fd8111b44a039515cb199e5 and https://github.com/optimad/bitpit/pull/434/commits/e8a8c600ffaaec28629801f1b529cec9165594c9.